shell/platform/fuchsia/dart-pkg/zircon/lib/src/handle.dart

```diff
@@ -17,33 +17,40 @@ class Handle extends NativeFieldWrapperClass1 {
   factory Handle.invalid() {
     return _createInvalid();
   }
-  static Handle _createInvalid() native 'Handle_CreateInvalid';
+  @pragma('vm:external-name', 'Handle_CreateInvalid')
+  external static Handle _createInvalid();
 
-  int get handle native 'Handle_handle';
+  @pragma('vm:external-name', 'Handle_handle')
+  external int get handle;
 
-  int get koid native 'Handle_koid';
+  @pragma('vm:external-name', 'Handle_koid')
+  external int get koid;
 
   @override
   String toString() => 'Handle($handle)';
 
   @override
   bool operator ==(Object other) {
-    return other is Handle
-        && other.handle == handle;
+    return other is Handle &&
```

Comment:
spurious formatting?

Reply:
Yeah, I changed the code, it autoformatted, then I had to undo the autoformatting by-hand, ...

(Flutter doesn't use `dart format` for reasons beyond my comprehension)

Reply:
> (Flutter doesn't use `dart format` for reasons beyond my comprehension)

++